src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -1,19 +1,208 @@
 <template>
-    <p>Run workflow progress is coming soon</p>
+    <div
+        class="container mt-3"
+        id="workflow-progress-container"
+        v-if="workflowRunSummaries"
+    >
+        <div class="row mb-3">
+            <label for="workflows" class="form-label col">Workflow</label>
+            <div class="col-10 px-0">
+                <v-select
+                    :options="workflowRunSummaries.data"
+                    label="name"
+                    :reduce="(label) => label.key"
+                    name="workflows"
+                    id="v-select"
+                    v-model="selectedWorkflowKey"
+                    placeholder="Select workflow or search by name..."
+                >
+                    <template #option="{ name, date }">
+                        <div>
+                            {{ name }}
+                            <span style="opacity: 0.5; float: right">
+                                {{ formatDate(date) }}</span
+                            >
+                        </div>
+                    </template>
+                </v-select>
+            </div>
+        </div>
+        <div class="row" v-if="workflowRunStatus" id="workflow-table">
+            <label class="form-label col">Reports</label>
+            <table class="table-bordered col-10">
+                <tr v-for="report in workflowRunStatus.data.reports">
+                    <td v-if="report.status === 'success'" class="p-2">
+                        <a
+                            :href="
+                                reportVersionHref(report.name, report.version)
+                            "
+                            >{{ report.name }}</a
+                        >
+                    </td>
+                    <td v-else-if="report.status === 'error'" class="p-2">
+                        <a
+                            :href="reportLogsHref(report.name)"
+                            @click="setReportLogsTab"
+                            >{{ report.name }}</a
+                        >
+                    </td>
```

Comment:
I don't think there's value in hyperlinking this, given it doesn't point at anything. As noted in the mockups, exactly how we display failure logs is an open question. Logged as mrc-2514.